src/lib.rs

```diff
@@ -176,20 +176,19 @@ impl<C: Config> Default for GgrsPlugin<C> {
 
 impl<C: Config> Plugin for GgrsPlugin<C> {
     fn build(&self, app: &mut App) {
-        let mut schedule = Schedule::default();
-        schedule.set_build_settings(ScheduleBuildSettings {
-            ambiguity_detection: LogLevel::Error,
-            ..default()
-        });
-
         app.init_resource::<RollbackFrameCount>()
             .init_resource::<ConfirmedFrameCount>()
             .init_resource::<MaxPredictionWindow>()
             .init_resource::<RollbackOrdered>()
             .init_resource::<LocalPlayers>()
             .init_resource::<FixedTimestepData>()
-            .add_schedule(GgrsSchedule, schedule)
-            .add_schedule(ReadInputs, Schedule::new())
+            .edit_schedule(GgrsSchedule, |schedule| {
```

Comment:
How is this change related to updating to bevy 0.12? I see why it is nice to have ambiguity being an error, but that is an issue separate from updating to 0.12, if I understand correctly.

Reply:
I think it was just done further up in the file before?

Reply:
Oh, I see! silly me :)